common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceMinificationService.java

```diff
@@ -56,9 +75,11 @@
      * Indicates whether or not the system is allowed to attempt to minify individual files. This can be useful if
      * the YUI compressor is failing to minify JavaScript/CSS due to syntax errors and you are attempting to track
      * down which file is problematic. It should not be enabled in a production environment.
-     * 
+     *
+     * @deprecated
      * @return the value of the system property "minify.allowSingleMinification"
      */
+    @Deprecated
     boolean getAllowSingleMinification();
 
```

Comment:
We probably need to make sure that it's fine but I would rather have this method just moved to the YUI compressor implementation. I don't know why this was on the interface in the first place.